rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -321,68 +318,24 @@ bool ppu_interpreter::VADDFP(ppu_thread& ppu, ppu_opcode_t op)
 	return true;
 }
 
-bool ppu_interpreter::VADDSBS(ppu_thread& ppu, ppu_opcode_t op)
```

Comment:
I would prefer to avoid moving the implementation. Put precise at the bottom of fast implementations.